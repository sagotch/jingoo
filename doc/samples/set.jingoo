{%- set hoge = "ok" -%}
{%- set foo, bar = ("foo", "bar") -%}
hoge: {{ hoge }}
foo: {{ foo }}
bar: {{ bar }}

{% set iterated = false -%}
{%- for item in [ 1, 2, 3 ] -%}{% set iterated = true %}{% endfor -%}
{%- if not iterated %}Did not iterate!{% endif %}
