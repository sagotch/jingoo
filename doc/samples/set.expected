hoge: ok
foo: foo
bar: bar

Did not iterate!
